translations/ui-inventory/en.json

```diff
@@ -436,8 +436,8 @@
   "item.status.orderClosed": "Order closed",
   "inTransitReport": "In transit items report (CSV)",
   "saveInstancesUIIDS": "Save instances UUIDs",
-  "exportInstancesMARC": "Export instances (MARC)",
-  "exportInstancesJSON": "Export instances (JSON)",
+  "exportInstancesMrc": "Export instances (MARC)",
+  "exportInstancesJson": "Export instances (JSON)",
```

Comment:
Let's change it to be `exportInstancesInMARC` and `exportInstancesInJSON`

Reply:
ok